Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Powershell_Command_Execution.yml

```diff
@@ -0,0 +1,1481 @@
+id: silent-Suspicious Powershell Command Execution
+version: -1
+issilent: true
+name: silent-Suspicious Powershell Command Execution
+description: |-
+  This playbook handles the following alerts:
+
+  - PowerShell runs suspicious base64-encoded commands
+  - PowerShell suspicious flags
+  - Reverse shell using PowerShell
+  - PowerShell downloads files via BITS
+
+  Playbook Stages
+
+  Analysis:
+
+  - Analyze command-line for suspicious patterns
+  - Evaluate risk score of PowerShell commands
+  - Check for indicators of compromise (IOCs)
+    - If suspicious/malicious patterns detected:
+      - Execute Remediation
+
+  Investigation:
+
+  - Check process reputation
+  - Analyze process causality chain
+  - Check for related alerts
+  - Search for suspicious execution patterns
+    - If evidence of malicious activity found:
+      - Execute Remediation
+
+  Containment:
+
+  - Terminate causality process tree
```

Comment:
- Terminate the causality process tree.

Question: is this the order containment is performed? If so, should be a numbered list